### PR TITLE
DEVX-3 convert to common structure following VITE changes

### PR DESCRIPTION
**Pull request by ts-ron**

### CODE REVIEW

File: login-with-email/backend/index.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
-const express = require('express');
-const axios = require('axios');
-const qs = require('qs');
+import express  from 'express';
+import axios from 'axios';
```
Comment (HilaPartukTS) [inline login-with-email/backend/index.js:2]: the axiosstill needs to be removed, right?

File: login-with-email/package.json [hunk 2]
```diff
@@ -3,15 +3,27 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "nodemon -r dotenv/config app.js localhost 8080"
+    "start": "vite --debug --host --port 8080",
```
Comment (HilaPartukTS) [inline login-with-email/package.json:6]: why this change?

File: login-with-email/pages/email-otp.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <link rel="stylesheet" type="text/css" href="stylesheets/dev-x-style.css">
+    <link rel="stylesheet" type="text/css" href="/stylesheets/dev-x-style.css">
```
Comment (HilaPartukTS) [inline login-with-email/pages/email-otp.html:4]: this needs to be changed to the new css file? 
AKA style.css

Comment (ts-ron) [reply login-with-email/pages/email-otp.html:4]: Yes, defering this to another PR, wanted this one to focus on the VITE change. Note this is a PR into your branch, not into main.

File: login-with-email/package.json (hunk 2 above)
Comment (ts-ron) [reply login-with-email/package.json:6]: This is part of the VITE dev (hot reload), it still launches our express code

File: login-with-email/backend/index.js (hunk 1 above)
Comment (ts-ron) [reply login-with-email/backend/index.js:2]: Yes, defering this to another PR, wanted this one to focus on the VITE change. Note this is a PR into your branch, not into main.

**Review by HilaPartukTS — APPROVED** [review]
